app/config/ApplicationConfig.scala

```diff
@@ -50,6 +50,8 @@ class ApplicationConfig @Inject()(config: ServicesConfig, configuration: Configu
 
   lazy val ssoUrl = Some(getConf("portal.ssoUrl"))
 
+  lazy val navTitleLink = "/annual-tax-summary/"
```

Comment:
There's a value in application.conf called `accessibility-statement.service-path` that already contains this which you can retrieve with `getConf("accessibility-statement.service-path")`

There's also `sa.language.fallbackUrl`, which wouldn't work as well because of confusion the name could cause. Not sure why we have duplicates of the same Url

Reply:
yes you are right @Christian-QA I saw sa.language.fallbackUrl too but didnt think it would make sense to someone that might be looking at main template and thinking why are passing sa.language.fallbackUrl in nav title
@pascal22p whats the best way forward? should we have a generic name that works for all?

Reply:
i have cleaned up app config and made the val generic so it can be used again